draft-js-plugins-editor/src/Editor/index.js

```diff
@@ -219,8 +153,53 @@ export default class PluginEditor extends Component {
     this.refs.editor.focus();
   };
 
+  createHandleListener = (name) => (event) => (
+    this.plugins
+      .filter((plug) => plug[name])
+      .map((plugin) => plugin[name](event))
+      .find((result) => result === true) === true
+  );
+
+  createOnListener = (name) => (event) => (
+    this.plugins
+      .filter(plug => typeof plug[name] === 'function')
+      .forEach(plug => plug[name](event))
+  );
+
+  createEventListeners = () => {
+    const listeners = {
+      onChange: this.onChange,
+      editorState: this.editorState,
```

Comment:
Doesn't make sense for `editorState` to be in the listeners section since it's just data.